consensus/src/consensus_observer/observer/consensus_observer.rs

```diff
@@ -1232,3 +1226,45 @@ fn update_metrics_for_ordered_block_message(
         ordered_block.proof_block_info().round(),
     );
 }
+
+/// Increments the dropped message counter for the given peer and message
+fn update_dropped_message_counter(peer_network_id: &PeerNetworkId, message_label: &str) {
```

Comment:
nit: should these functions be `increment_` instead of `update_`?

Reply:
SGTM. Will update 😄